meganav.yaml

```diff
@@ -34,5 +724,175 @@ get_ubuntu:
               url: /tutorials/install-ubuntu-desktop#1-overview
             - title: Try before installation
               url: /tutorials/try-ubuntu-before-you-install#1-getting-started
+            - title: Case studies, whitepapers & webinars
+              url: /engage
             - title: All tutorials
               url: /tutorials
```

Comment:
```suggestion
              url: /tutorials
        section_footer:
        cta_title: Download 22.04.1 LTS
        cta_url: /download/desktop/thank-you?version=20.04.3&architecture=amd64
```

Reply:
Sorry - more context for this one: the design was updated to move what used to be in the footer (the download CTA) to the body of one of the items above (Ubuntu Desktop). The link did need updating though, I'm doing that now.